src/main/java/racingcar/domain/Cars.java

```diff
@@ -2,30 +2,46 @@
 
 import java.util.List;
 import java.util.stream.Collectors;
+import racingcar.domain.movable.MovableStrategy;
+import racingcar.utils.Parser;
 
 public class Cars {
 
     private final List<Car> cars;
 
-    public Cars(final List<Car> cars) {
+    private Cars(final List<Car> cars) {
         this.cars = cars;
     }
 
+    public static Cars fromString (final String carNames) {
+        return new Cars(Parser
+            .parseCarNames(carNames)
+            .stream()
+            .map(Car::new)
+            .collect(Collectors.toList()));
```

Comment:
한 라인에 모든 결과를 담으려하면 오히려 가독성이 떨어질 수 있어요!
아래와 같이 분리해보면 어떨까요?

```suggestion
        final List<Car> cars = Parser.parseCarNames(carNames).stream()
            .map(Car::new)
            .collect(Collectors.toList());
        return new Cars(cars);
```